web/sites/default/config/views.view.sitewide_groups_index.yml

```diff
@@ -287,6 +243,90 @@ display:
             default_group: All
             default_group_multiple: {  }
             group_items: {  }
+        search_api_language:
+          id: search_api_language
+          table: search_api_index_full_index
+          field: search_api_language
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: search_api_language
+          operator: in
+          value:
+            '***LANGUAGE_language_interface***': '***LANGUAGE_language_interface***'
+          group: 1
+          exposed: false
+          expose:
+            operator_id: ''
+            label: ''
+            description: ''
+            use_operator: false
+            operator: ''
+            operator_limit_selection: false
+            operator_list: {  }
+            identifier: ''
+            required: false
+            remember: false
+            multiple: false
+            remember_roles:
+              authenticated: authenticated
+            reduce: false
+          is_grouped: false
+          group_info:
+            label: ''
+            description: ''
+            identifier: ''
+            optional: true
+            widget: select
+            multiple: false
+            remember: false
+            default_group: All
+            default_group_multiple: {  }
+            group_items: {  }
+        group_subtype:
+          id: group_subtype
```

Comment:
Not related to this code, but I didn't have a better place to put it:

Could you update the `views.view.displayed_in.yml` : `entity_reference_council_document_department` view to filter on the new group subtype taxonomy instead of the actual group type? This is the only view I noticed was missed.

You can set it to Bureau/Office and Program only, as council clerk actually just recently requested we restrict it to those.

Reply:
Merged with the most recent Master head and updated the filter in that view.